server/server.py

```diff
@@ -150,15 +161,15 @@ def readDB(graph, issueTypes, creationFromDate = None, creationToDate = None, re
     totCommentsPerIssue = pd.DataFrame(graph.data(query))
 
     # Organizational affiliation per username
-    query = """MATCH (n:User) RETURN n.key AS author, n.organization AS organization"""
+    query = """MATCH (n:User) WHERE NOT(n.ignore) RETURN n.key AS author, n.organization AS organization"""
 
     orgData = pd.DataFrame(graph.data(query))
 
     # Add column with total number of comments per issue to issueData
     issueData = pd.merge(issueData, totCommentsPerIssue, how="right", on="issueId")
     #
     # Add column with organizational affiliation
-    issueData = pd.merge(issueData, orgData, how="outer", on="author")
+    issueData = pd.merge(issueData, orgData, how="inner", on="author")
```

Comment:
Is this an OK change to make? I want to eliminate users that are not in orgData from issueData.